fastlane_bot/helpers/txhelpers.py

```diff
@@ -117,578 +52,129 @@ def validate_and_submit_transaction(
-        The following condition must be met:
-        maxFeePerGas <= baseFee + maxPriorityFeePerGas
-
-        returns: the transaction to be submitted to the blockchain
-        """
-        max_priority_fee = int(max_priority_fee)
-        base_gas_price = int(gas_price)
-        max_gas_price = base_gas_price + max_priority_fee
-
-        if self.ConfigObj.NETWORK == self.ConfigObj.NETWORK_TENDERLY:
-            self.wallet_address = self.ConfigObj.BINANCE14_WALLET_ADDRESS
-            
-        # if "tenderly" in self.web3.provider.endpoint_uri:
-        #     print("Tenderly network detected: Manually setting maxFeePerFas and maxPriorityFeePerGas")
-        #     max_gas_price = 3
-        #     max_priority_fee = 3
-
-        if self.ConfigObj.NETWORK in ["ethereum", "coinbase_base"]:
-            tx_details = {
-                "type": "0x2",
-                "maxFeePerGas": max_gas_price,
-                "maxPriorityFeePerGas": max_priority_fee,
-                "from": self.wallet_address,
-                "nonce": nonce,
-            }
-        else:
-            tx_details =  {
-                "gasPrice": max_gas_price,
-                "from": self.wallet_address,
-                "nonce": nonce,
-            }
-        tx_details["value"] = value
-        return tx_details
-
-    def submit_regular_transaction(self, signed_tx) -> str:
-        """
-        Submits the transaction to the blockchain.
-
-        :param signed_tx: the signed transaction to be submitted to the blockchain
-
-        returns: the transaction hash of the submitted transaction
-        """
-
-        self.ConfigObj.logger.info(
-            f"[helpers.txhelpers.submit_regular_transaction] Attempting to submit transaction {signed_tx}"
-        )
-
-        return self._submit_transaction(self.web3.eth.send_raw_transaction(signed_tx.rawTransaction))
-
-    def submit_private_transaction(self, signed_tx, block_number: int) -> str:
-        """
-        Submits the transaction privately through Alchemy -> Flashbots RPC to mitigate frontrunning.
-
-        :param signed_tx: the signed transaction to be submitted to the blockchain
-        :param block_number: the current block number
-
-        returns: The transaction receipt, or None if the transaction failed
-        """
-
-        self.ConfigObj.logger.info(
-            f"[helpers.txhelpers.submit_private_transaction] Attempting to submit transaction to Flashbots"
-        )
-
-        params = [
-            {
-                "tx": signed_tx.rawTransaction.hex(),
-                "maxBlockNumber": hex(block_number + 10),
-                "preferences": {"fast": True},
-            }
-        ]
 
+    def _send_private_transaction(self, raw_tx):
         response = self.alchemy.core.provider.make_request(
             method="eth_sendPrivateTransaction",
-            params=params,
+            params=[{"tx": raw_tx, "maxBlockNumber": self.cfg.w3.eth.block_number + 10, "preferences": {"fast": True}}],
```

Comment:
tx & maxBlockNumber are submitted here as hex. See original function.
![image](https://github.com/bancorprotocol/fastlane-bot/assets/110377282/84225c8b-fc75-4083-9d4f-5863b4befe73)

Reply:
It doesn't need to be.
A 10-base string, a 16-base string starting with "0x", and a native number are all fine.

Reply:
Native number is not fine according to documentation.

Reply:
Fixed, TX!